packages/core/src/components/side-pane/side-pane.tsx

```diff
@@ -105,6 +111,11 @@ export class SidePane {
     if (this.expandPane) {
       this.onExpandedChange();
     }
+
+    applicationLayoutService.setBreakpoints(this.breakpoints);
```

Comment:
application Layout Service is a singleton, do not set Breakpoints this effects hole application.